Update cloudfront_distribution.html.markdown (#29015)
hashicorp/terraform-provider-aws: website/docs/r/cloudfront_distribution.html.markdown

    response to requests using the specified HTTP methods.

* `cache_policy_id` (Optional) - The unique identifier of the cache policy that

>     is attached to the cache behavior. Either [`cache_policy_id` and optional `origin_request_policy_id`] or

Reviewer: To provide some clarity, the CloudFront Distributions are required to have a DefaultCacheBehavoir, and optionally can have CacheBehaviors specified (via the schema attribute `ordered_cache_behavoir`).

When configuring the DefaultCacheBehaoir the user must include either a CachePolicyId or ForwardedValues, and is separate from speciying OriginRequestPolicyId.

```suggestion
    is attached to the cache behavior. If configuring the `default_cache_behavior` either `cache_policy_id` or `forwarded_values` must be set.
```

>     `forwarded_values` must be set.

Reviewer: Suggested to remove.
```suggestion

```

* `field_level_encryption_id` (Optional) - Field level encryption configuration ID

> * `forwarded_values` (Optional **Deprecated** use `cache_policy_id` or `origin_request_policy_id ` instead) - The [forwarded values configuration](#forwarded-values-arguments) that specifies how CloudFront

Reviewer 2: Looks like this last part of this line may have gotten chopped off.

```suggestion
* `forwarded_values` (Optional **Deprecated** use `cache_policy_id` or `origin_request_policy_id ` instead) - The [forwarded values configuration](#forwarded-values-arguments) that specifies how CloudFront handles query strings, cookies and headers (maximum one).
```

* `field_level_encryption_id` (Optional) - Field level encryption configuration ID

> * `forwarded_values` (Optional) - The [forwarded values configuration](#forwarded-values-arguments) that specifies how CloudFront

Reviewer: ```suggestion
* `forwarded_values` (Optional **Deprecated** use `cache_policy_id` or `origin_request_policy_id ` instead) - The [forwarded values configuration](#forwarded-values-arguments) that specifies how CloudFront
```

>     handles query strings, cookies and headers (maximum one). Either [`cache_policy_id` and optional `origin_request_policy_id`] or

Reviewer: Suggested to remove.

```suggestion
```

>     `forwarded_values` must be set. Note, AWS recommend that you use a cache policy or an 

Reviewer: Suggested to remove.

```suggestion
```

>     origin request policy instead of this field.

Reviewer: Suggested to remove.

```suggestion
```

    whether the object has been updated. Defaults to 0 seconds.

* `origin_request_policy_id` (Optional) - The unique identifier of the origin request policy

>     that is attached to the behavior.

Reviewer 2: Was this intentionally removed? Due to being unfamiliar, I can't quite tell if this bit of the sentence is important, or was removed because it's not.

    whether the object has been updated. Defaults to 0 seconds.

* `origin_request_policy_id` (Optional) - The unique identifier of the origin request policy

>     that is attached to the behavior. Either [`cache_policy_id` and optional `origin_request_policy_id`] or

Reviewer: Suggested to remove.

```suggestion
```

>     `forwarded_values` must be set.

Reviewer: Suggested to remove.

```suggestion
```
